### DISCUSSION TITLE
Cannot add Tavily Search Plugin to agent

### DISCUSSION THREAD

**Discussion by loopdeloop76** [danny-avila/LibreChat#4665] [Troubleshooting]
### What happened?

I have enabled the experimental agents (which look really promising!).
In the agent builder I am unable to add the Tavily Search Plugin.

### Steps to Reproduce

1. Enable agents with env variable `EXPERIMENTAL_AGENTS=true`
2. In Agent Builder click on Add Tools
3. Click on Tavily Search -> Add
4. Set Tavily API Key and save
5. Nothing happens (no tool added)

### What browsers are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Fixing here: [🫧 fix: Tool Auth Form Button to Prevent Form Bubbling](https://github.com/danny-avila/LibreChat/pull/4666)
